src/aurora/actionscript/avm.cpp

```diff
@@ -41,6 +41,9 @@ namespace ActionScript {
 
 AVM::AVM() {
 	_registers.resize(256);
+	for (int i = 0; i < 256; ++i)
```

Comment:
Please use _registers.size() here (and size_t i), or iterators, or a range-based for, so that we're not repeating the 256 here.